src/internal/m365/collection/drive/permission.go

```diff
@@ -390,7 +390,12 @@ func UpdateLinkShares(
 		}
 
 		if err != nil {
-			el.AddRecoverable(ctx, clues.Stack(err))
+			if graph.IsErrSharingDisabled(err) {
+				logger.Ctx(ctx).Info("unable to restore link share; sharing disabled")
```

Comment:
Nit: no harm in including the error, even at an info level.
```suggestion
				logger.CtxErr(ctx, err).Info("unable to restore link share; sharing disabled")
```

Reply:
Also, should this use `ictx`?  I see that in use on line 388.